x/ibc/03-connection/keeper/keeper.go

```diff
@@ -94,7 +94,7 @@ func (k Keeper) SetClientConnectionPaths(ctx sdk.Context, clientID string, paths
 func (k Keeper) addConnectionToClient(ctx sdk.Context, clientID, connectionID string) sdk.Error {
 	conns, found := k.GetClientConnectionPaths(ctx, clientID)
 	if !found {
-		return types.ErrClientConnectionPathsNotFound(k.codespace, clientID)
+		conns = []string{}
```

Comment:
I don't understand this change

Reply:
it store all the connection path under a particular client, so if `addConnectionToClient` for the first time, the `conns` is empty.

Reply:
Already fixed in the upstream branch.